tests/testdata/codegen/transpiled_examples/cue-random-pp/dotnet/dotnet.csproj

```diff
@@ -2,12 +2,13 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>

```

Comment:
Hmm I am just now noticing this diff (and other like this one below). I did rebase with the latest from `master`.

Reply:
Ah, probably because I only have dotnet SDK 6.0 installed on my machine?